master/internal/experiment.go

```diff
@@ -279,8 +279,15 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 
 	// Experiment shutdown logic.
 	case actor.PostStop:
-		if err := e.db.SaveExperimentProgress(e.ID, nil); err != nil {
-			ctx.Log().Error(err)
+		if e.State != model.CompletedState {
```

Comment:
if thought we would want to set this to 1.0 if it _was_ completed. and that otherwise we would not set it at all (with the thought that, if an experiment fails at 12% progress, it is much more useful to see 12% progress than nil).

Reply:
crazy github glitch here